hobo/script/extract_hobo.py

```diff
@@ -97,7 +97,7 @@ def get_csv_from_folder_not_in_db(conn, csv_filename):
     earliest_csv_timestamp = pendulum.instance(csv_readings.iloc[0]['Date Time, GMT-10:00'], 'Pacific/Honolulu')
     latest_csv_timestamp = pendulum.instance(csv_readings.iloc[csv_readings.shape[0]-1]['Date Time, GMT-10:00'], 'Pacific/Honolulu')
     for sensor_info_row in sensor_info_rows:
-        error_log_row = orm_hobo.ErrorLog(status=True, purpose_id=sensor_info_row.purpose_id, datetime=current_time, pipeline_stage=orm_hobo.PipelineStage.data_acquisition)
+        error_log_row = orm_hobo.ErrorLog(status=True, purpose_id=sensor_info_row.purpose_id, datetime=current_time, pipeline_stage=orm_hobo.ErrorLog.PipelineStageEnum.data_acquisition)
```

Comment:
Here's an example of a row where we set the value pipeline_stage using the PipelineStageEnum class.

If I remove the class, I might have to set this value using something like a string 'data_acquisition'.

Reply:
I'm confused on the apparition of this ErrorLog class. Maybe it's best we just discuss this on call.

Reply:
Ok per the call I was reminded we agreed on moving 2 enum classes to inside their tables because they are associated with columns in these tables.